docs/autodidax.ipynb

```diff
@@ -1423,6 +1423,18 @@
   "kernelspec": {
    "display_name": "Python 3",
    "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6"
```

Comment:
Not sure you meant to edit this file?

Reply:
These changes are in the first commit - fixing some build warnings that were causing local failures for me.